packages/beacon-node/src/api/impl/lodestar/index.ts

```diff
@@ -210,10 +210,12 @@ export function getLodestarApi({
       const proof = new Tree(stateView.node).getSingleProof(gindex);
 
       return {
-        data: {
-          historicalSummaries: stateView.historicalSummaries.toValue(),
-          proof: proof,
+          data: {
+            slot: stateView.slot,
+            historicalSummaries: stateView.historicalSummaries.toValue(),
+            proof: proof,
         },
+        meta: {executionOptimistic, finalized, version: config.getForkName(stateView.slot)},
```

Comment:
we already go the fork above

```suggestion
        meta: {executionOptimistic, finalized, version: fork},
```